bears/python/PyDocStyleBear.py

```diff
@@ -4,7 +4,6 @@
 
 
 @linter(executable='pydocstyle',
-        use_stdout=False,
         use_stderr=True,
```

Comment:
@jayvdb Yeah, I tried that. If it is changed we are getting error in the output.